Descent3/LoadLevel.cpp

```diff
@@ -3588,6 +3588,86 @@ static inline char* GetCurrentSumString() {
   return output_buf;
 }
 
+bool GetLevelInfo(const std::filesystem::path &filename, level_info &info) {
```

Comment:
`GetLevelInfo` is also a DMFC function with a different signature that returns a `level_info` object. I would suggest renaming this function to avoid possible confusion to something like `LoadLevelInfo`, as this function loads and retrieves the level info from file.